localisation/00_wc_sad.csv

```diff
@@ -73,34 +71,29 @@ learn_talon_form_desc;Learn how to shapeshift into a great bird.;;Lernt, Euch in
 learn_stag_form;Learn Stag Form;;Lernt die Hirschform;;;;;;;;;;;x
 learn_stag_form_desc;Learn how to shapeshift into a great stag.;;Lernt, Euch in einen Hirsch zu verwandeln.;;;;;;;;;;;x
 learn_master_cat_form;Master Cat Form;;Meistert die Katzenform;;;;;;;;;;;x
-learn_master_cat_form_desc;Attain mastery over the Cat Form.;;Erlangt Meisterschaft �ber die Katzenform.;;;;;;;;;;;x
+learn_master_cat_form_desc;Attain mastery over the cat form.;;Erlangt Meisterschaft �ber die Katzenform.;;;;;;;;;;;x
 learn_master_bear_form;Master Bear Form;;Meistert die B�renform;;;;;;;;;;;x
-learn_master_bear_form_desc;Attain mastery over the Bear Form.;;Erlangt Meisterschaft �ber die B�renform.;;;;;;;;;;;x
+learn_master_bear_form_desc;Attain mastery over the bear form.;;Erlangt Meisterschaft �ber die B�renform.;;;;;;;;;;;x
 learn_master_talon_form;Master Talon Form;;Meistert die Klauenform;;;;;;;;;;;x
-learn_master_talon_form_desc;Attain mastery over the Talon Form.;;Erlangt Meisterschaft �ber die Klauenform.;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+learn_master_talon_form_desc;Attain mastery over the talon form.;;Erlangt Meisterschaft �ber die Klauenform.;;;;;;;;;;;x
 cat_form_druid;Cat Form;;Katzenform;;;;;;;;;;;x
-cat_form_druid_desc;Shapeshift into Cat Form.;;Verwandelt Euch in eine Gro�katze.;;;;;;;;;;;x
+cat_form_druid_desc;Shapeshift into the cat form.;;Verwandelt Euch in eine Gro�katze.;;;;;;;;;;;x
 bear_form_druid;Bear Form;;B�renform;;;;;;;;;;;x
-bear_form_druid_desc;Shapeshift into Bear Form.;;Verwandelt Euch in einen B�ren.;;;;;;;;;;;x
+bear_form_druid_desc;Shapeshift into the bear form.;;Verwandelt Euch in einen B�ren.;;;;;;;;;;;x
 talon_form_druid;Talon Form;;Klauenform;;;;;;;;;;;x
-talon_form_druid_desc;Shapeshift into Talon Form.;;Verwandelt Euch in einen Falken.;;;;;;;;;;;x
+talon_form_druid_desc;Shapeshift into the talon form.;;Verwandelt Euch in einen Falken.;;;;;;;;;;;x
 stag_form_druid;Stag Form;;Hirschform;;;;;;;;;;;x
-stag_form_druid_desc;Shapeshift into Stag Form.;;Verwandelt Euch in einen Hirsch.;;;;;;;;;;;x
+stag_form_druid_desc;Shapeshift into the stag form.;;Verwandelt Euch in einen Hirsch.;;;;;;;;;;;x
```

Comment:
I think druid forms should be upper case

Reply:
done